tools/update_config_version.py

```diff
@@ -0,0 +1,87 @@
+"""
+Script to find and update all armory versions in JSON configs to current version
```

Comment:
In reviewing this pull request, I thought "gee, hitting all those json files is tedious and error prone. I should have expected the existence of this script based on the PR author.